envoy/adapter/adapter.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/golang/protobuf/ptypes/duration"
 	"github.com/golang/protobuf/ptypes/wrappers"
 	log "github.com/sirupsen/logrus"
+	anypb "google.golang.org/protobuf/types/known/anypb"
```

Comment:
```suggestion
	"google.golang.org/protobuf/types/known/anypb"
```

Reply:
Sadly it requires the name. The internal name of the package is something wacky

Reply:
Yeah, "thanks" Google...